simple.c

```diff
@@ -217,6 +217,8 @@ bool Simple_Invoke(void *SimpleContext, echandle MethodDictionaryHandle, echandl
     } else if (strcmp(Method, "startValueRequest") == 0) {
         RetVal = Simple_StartValueRequest(Simple);
         IDictionary_AddInt(ReturnDictionaryHandle, "returnValue", RetVal, &ItemHandle);
+    } else {
+        RetVal = false;
```

Comment:
observation/nit: I never liked the idea of defaulting to a successful value. In this case it's probably less fragile than trying to set `RetVal = true` in all the accessors.